### DISCUSSION TITLE
use renovate

### DISCUSSION THREAD

**Discussion by ToBinio** [modrinth/.github#183] [Other discussions]
I believe it would be beneficial to use [renovate](https://docs.renovatebot.com/) or something like it.

in case somebody does not know what renovate is:
it's a bot that automatically makes PR in case a project dependency is outdated and updates it in the PR

thoughts?

**Comment by brawaru** [comment]
Renovate is deffo worth having, but needs to be configured to not produce too much noise and not cause unintended changes (e.g., updating peers without solid reason). Tests would be nice too to avoid hidden regressions. Also TypeScript support needs to improve to statically type-check API use.
